dev-libs/immer/immer-0.8.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A library of persistent and immutable data structures written in C++"
+HOMEPAGE="https://sinusoid.es/immer/"
+SRC_URI="https://github.com/arximboldi/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Boost-1.0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE=""
+#RESTRICT="!test? ( test )"
+SLOT=0
+
+RDEPEND="
+dev-libs/boost
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+"
+
+src_configure() {
+
+	local mycmakeargs=(
+		-Dimmer_BUILD_DOCS=OFF
+		-Dimmer_BUILD_EXAMPLES=OFF
+		-Dimmer_BUILD_EXTRAS=OFF
+		-Dimmer_BUILD_TESTS=OFF
```

Comment:
no tests?

Reply:
The tests require an other new dep: https://github.com/catchorg/Catch2
I didn't want to add even more libraries to this PR, but I could try to hook this up as well if you insist.

Reply:
That seems to be `dev-cpp/catch`, actually.